tests/mmm-admin.spec.ts

```diff
@@ -188,4 +189,430 @@ describe('mmm-admin', () => {
       assert.deepEqual(poolAccountInfo.allowlists, allowlists);
     });
   });
+
+  describe('Can update allowlists', () => {
+    it('happy path', async () => {
+      const fvca = Keypair.generate();
+      const newFcva = Keypair.generate();
+
+      const allowlists = [
+        { kind: AllowlistKind.fvca, value: fvca.publicKey },
+        { kind: AllowlistKind.mint, value: cosigner.publicKey },
+        { kind: AllowlistKind.mcc, value: wallet.publicKey },
+        ...getEmptyAllowLists(3),
+      ];
+
+      const newAllowlists = [
+        { kind: AllowlistKind.fvca, value: newFcva.publicKey },
+        { kind: AllowlistKind.mint, value: cosigner.publicKey },
+        { kind: AllowlistKind.mcc, value: wallet.publicKey },
+        ...getEmptyAllowLists(3),
+      ];
+
+      const referral = Keypair.generate();
+      const uuid = Keypair.generate();
+
+      const { key: poolKey } = getMMMPoolPDA(
+        program.programId,
+        wallet.publicKey,
+        uuid.publicKey,
+      );
+
+      await program.methods
+        .createPool({
+          spotPrice: new anchor.BN(1 * LAMPORTS_PER_SOL),
+          curveType: CurveKind.linear,
+          curveDelta: new anchor.BN(0),
+          reinvestFulfillBuy: true,
+          reinvestFulfillSell: true,
+          expiry: new anchor.BN(42),
+          lpFeeBp: 200,
+          referral: referral.publicKey,
+          cosignerAnnotation: new Array(32).fill(0),
+          buysideCreatorRoyaltyBp: 0,
+
+          uuid: uuid.publicKey,
+          paymentMint: PublicKey.default,
+          allowlists,
+        })
+        .accountsStrict({
+          owner: wallet.publicKey,
+          cosigner: cosigner.publicKey,
+          pool: poolKey,
+          systemProgram: SystemProgram.programId,
+        })
+        .signers([cosigner])
+        .rpc();
+
+      await program.methods
+        .updateAllowlists({
+          allowlists: newAllowlists,
+        })
+        .accountsStrict({
+          cosigner: cosigner.publicKey,
+          owner: wallet.publicKey,
+          pool: poolKey,
+        })
+        .signers([cosigner])
+        .rpc();
+
+      const poolAccountInfo = await program.account.pool.fetch(poolKey);
```

Comment:
very nit - can we check that none of the other pool fields have changed from before the transaction?